Source/Shared/Storage/HybridStorage.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+class HybridStorage {
```

Comment:
Haven't we decided to get rid of hybrid cache?

Reply:
This is used internally inside `Storage`, just in case user wants 2 cache layers. If if used to offset some work from the `Storage`